finalProject/tenSpeckledFrogs.js

```diff
@@ -0,0 +1,11 @@
+var frogs = new Array(10); 
+
+for (var i = 10; i > 0; i--) {
+  if ([i] > 2) {
```

Comment:
Bracket notation is used to get an element by index from an array. Since you are just talking about the single variable i, you do not need brackets and can just use `i`

Reply:
makes sense, updated.